msal/src/main/java/com/microsoft/identity/client/PublicClientApplicationConfiguration.java

```diff
@@ -597,6 +602,14 @@ public void checkIntentFilterAddedToAppManifestForBrokerFlow() throws MsalClient
             return;
         }
 
+        if (mAppContext.getPackageName().equalsIgnoreCase(AuthenticationConstants.Broker.AZURE_AUTHENTICATOR_APP_PACKAGE_NAME))
```

Comment:
Should this go into isBrokerRedirectUri()?

Reply:
Do we want to try and gate this on the active broker being CP?

Reply:
I think the purpose of isBrokerRedirectUri() is to check the schema and then set/unset if the Broker should be used.
As this is a specific case I think we should keep it separate.